### PR TITLE
Update walkthrough-publishing-a-visual-studio-extension.md

### PR DESCRIPTION
**Pull request by amit-kulkarni-in**
Updating the documentation to provide steps to add the members to the publisher account. The new steps provide extension authors how to get the User Id and add to the member list.

### CODE REVIEW

**Comment by Court72** [comment]
@leslierichardson95 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md [hunk 1]
```diff
@@ -179,13 +179,18 @@ Before you begin, make sure that you've built the new release version of your ex
 
 Visual Studio Marketplace supports granting additional users permissions to access and manage a publisher account.
 
-1. Navigate to the publisher account you wish to add additional users to.
+1. Ask member (that you want to add) to login their account on Marketplace and get the User Id. User Id will be visible by hovering over email. As shown in the image.
```
Comment (ghogen) [inline docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md:182]:
```suggestion
1. Ask the member that you want to add to sign in to their account on Marketplace and get the User Id. The User Id will be visible by hovering over email, as shown in the following image:
```

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md [hunk 2]
```diff
@@ -179,13 +179,18 @@ Before you begin, make sure that you've built the new release version of your ex
...
+![Screenshot that shows the user ID next to a user's name and email address in Marketplace.](media/marketplace-user-id.png)
+
+2.	Member can copy the User Id by clicking copy-to-clipboard button and share with the current owner of the publisher (You).
```
Comment (ghogen) [inline docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md:185]:
```suggestion
2.	The member can copy the User Id by clicking the copy-to-clipboard button, and then share it with you.
```

**Review by ghogen — APPROVED** [review]

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md [hunk 3]
```diff
@@ -179,13 +179,18 @@ Before you begin, make sure that you've built the new release version of your ex
 
 Visual Studio Marketplace supports granting additional users permissions to access and manage a publisher account.
 
-1. Navigate to the publisher account you wish to add additional users to.
+1. Ask the member that you want to add to sign in to their account on Marketplace and get the User Id. The User Id will be visible by hovering over email, as shown in the following image:
+![Screenshot that shows the user ID next to a user's name and email address in Marketplace.](media/marketplace-user-id.png)
```
Comment (Jak-MS) [inline docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md:183]:
```suggestion
1. Ask the member that you want to add to sign in to their account on Marketplace and get the User Id. The User Id will be visible by hovering over email, as shown in the following image:

   ![Screenshot that shows the user ID next to a user's name and email address in Marketplace.](media/marketplace-user-id.png)
```

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md [hunk 4]
```diff
@@ -179,13 +179,19 @@ Before you begin, make sure that you've built the new release version of your ex
...
 
-1. Navigate to the publisher account you wish to add additional users to.
+1. Ask the member that you want to add to sign in to their account on Marketplace and get the User Id. The User Id will be visible by hovering over email, as shown in the following image:
 
-2. Select **Members** and click on **Add**.
+   ![Screenshot that shows the user ID next to a user's name and email address in Marketplace.](media/marketplace-user-id.png)
+
+2.	The member can copy the User Id by clicking the copy-to-clipboard button, and then share it with you.
+
+3.	Navigate to the publisher account you wish to add additional users to.
```
Comment (Jak-MS) [inline docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md:188]:
```suggestion
2. The member can copy the User ID by clicking the copy-to-clipboard button, and then share it with you.

3. Navigate to the publisher account you wish to add additional users to.
```

File: docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md [hunk 5]
```diff
@@ -179,13 +179,19 @@ Before you begin, make sure that you've built the new release version of your ex
 
 Visual Studio Marketplace supports granting additional users permissions to access and manage a publisher account.
 
-1. Navigate to the publisher account you wish to add additional users to.
+1. Ask the member that you want to add to sign in to their account on Marketplace and get the User Id. The User Id will be visible by hovering over email, as shown in the following image:
```
Comment (Jak-MS) [inline docs/extensibility/walkthrough-publishing-a-visual-studio-extension.md:182]:
```suggestion
1. Ask the member that you want to add to sign in to their account on Marketplace and get the User ID. The User ID will be visible by hovering over email, as shown in the following image:
```

**Pull request merged**